src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const titles = [...head.children[0].children];
+
+function sortTable(n) {
+  const table = document.querySelector('tbody');
+  let switching = true;
+  let rows, i, x, y, shouldSwitch;
```

Comment:
Rename all your boolean variables for more meaningful names. [Hint](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)